consul/plan.sh

```diff
@@ -0,0 +1,28 @@
+pkg_origin=core
+pkg_name=consul
+pkg_version=0.7.1
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("MPL-2.0")
+pkg_upstream=https://www.consul.io/
+pkg_source=https://releases.hashicorp.com/${pkg_name}/${pkg_version}/${pkg_name}_${pkg_version}_linux_amd64.zip
+pkg_shasum=5dbfc555352bded8a39c7a8bf28b5d7cf47dec493bc0496e21603c84dfe41b4b
+pkg_filename=${pkg_name}-${pkg_version}_linux_amd64.zip
+pkg_deps=()
+pkg_build_deps=(core/unzip)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_expose=(8400 8500 8600)
+pkg_svc_user=root
+
+do_unpack() {
+  cd "${HAB_CACHE_SRC_PATH}" || exit
+  unzip ${pkg_filename} -d "${pkg_name}-${pkg_version}"
+}
+
+do_build() {
+  return 0
+}
+
+do_install() {
+  install -D consul "${pkg_prefix}"/bin/consul
```

Comment:
I thought shellcheck wants this to be `"${pkg_prefix}/bin/consul"` - unless I'm wrong?